webknossos-datastore/app/com/scalableminds/webknossos/datastore/datavault/S3Utilities.scala

```diff
@@ -0,0 +1,83 @@
+package com.scalableminds.webknossos.datastore.datavault
+
+import com.amazonaws.auth.{
+  AWSCredentialsProvider,
+  AWSStaticCredentialsProvider,
+  AnonymousAWSCredentials,
+  BasicAWSCredentials
+}
+import com.amazonaws.regions.Regions
+import com.amazonaws.services.s3.{AmazonS3, AmazonS3ClientBuilder}
+
+import java.net.URI
+import java.util.Properties
+
+class S3Utilities {}
+
+class AnonymousAWSCredentialsProvider extends AWSCredentialsProvider {
+  override def getCredentials = new AnonymousAWSCredentials
+
+  override def refresh(): Unit = {}
+}
+
+object S3Utilities {
```

Comment:
I think both classes are small enough that it would be acceptable to move this stuff into S3DataVault directly (and make it private there), so that we don’t have to import all this there.